### PR TITLE
Fix typos in Hai Rescue: Ooonem

### PR DESCRIPTION
**Pull request by roadrunner56**
**Content (Artwork / Missions / Jobs)**

## Summary
Adds a missing 'you', and changes a pronoun from 'her' to 'them', as at that point the player is unable to tell what gender the Hai might be. 'She' is used once the player gets a good look at Ooonem.

### CODE REVIEW

**Review by EjoThims — APPROVED** [review]
I'm good with that transition and where it is.

**Review by bene-dictator — APPROVED** [review]

File: data/hai/hai reveal 2 start.txt [hunk 1]
```diff
@@ -246,10 +246,10 @@ mission "Hai Rescue: Ooonem"
...
 			label start
 			choice
-				`	(Talk to her.)`
+				`	(Talk to them.)`
 				`	(Come back later.)`
 					defer
-			`	As step into her path, she swings the laser rifle towards you and says, "You're either very brave, very stupid, or know something... I should warn you, I'm not going back to them!"`
+			`	As you step into her path, the Hai swings the laser rifle towards you and says, "You're either very brave, very stupid, or know something... I should warn you, I'm not going back to them!"`
```
Comment (warp-core) [inline data/hai/hai reveal 2 start.txt:252]: `As you step into *her* path,`

Do we know her gender at this point or not?
Also, the next line says `she`. Do we know it by that point?

Comment (roadrunner56) [reply data/hai/hai reveal 2 start.txt:252]: Yes?

Comment (MasterOfGrey) [reply data/hai/hai reveal 2 start.txt:252]: It's softly implied by the construction of this that it's apparent once she speaks to you, but doesn't bear remarking upon.

Comment (warp-core) [reply data/hai/hai reveal 2 start.txt:252]: You've changed the earlier reference to be gender neutral, and a reference after the part I quoted, (though, maybe that was just to remove hte pronoun altogether?) and she doesn't start speaking until the end of this line.

Comment (roadrunner56) [reply data/hai/hai reveal 2 start.txt]: I changed `As you step into her path` to `As you step into their path` to maintain consistency.

**Pull request merged**